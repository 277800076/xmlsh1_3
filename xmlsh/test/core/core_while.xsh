# core_while.xsh

while false ; do
  echo SHN 
done
# core_while.xsh
# tests of while

while false ; do echo SNH ; done

RET=1
echo before while
while [ -z "$X" ] ; do
   echo while RET = $RET
   X="A"
   RET=0
   echo while2 RET = $RET   
done

# Until
until true ; do
	echo Failed
	exit 1
done

X=0

until [ $X -eq 1 ] ; do
	echo Until $X
	X=1
done


echo RET = $RET
exit 0