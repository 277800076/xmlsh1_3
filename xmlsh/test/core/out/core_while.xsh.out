before while
Until 0
RET = 1
